chromadb/utils/embedding_functions/sentence_transformer_embedding_function.py

```diff
@@ -27,28 +28,85 @@ def __init__(
             normalize_embeddings (bool, optional): Whether to normalize returned vectors, defaults to False
             **kwargs: Additional arguments to pass to the SentenceTransformer model.
         """
+        try:
+            from sentence_transformers import SentenceTransformer
+        except ImportError:
+            raise ValueError(
+                "The sentence_transformers python package is not installed. Please install it with `pip install sentence_transformers`"
+            )
+
+        self.model_name = model_name
+        self.device = device
+        self.normalize_embeddings = normalize_embeddings
+        self.kwargs = kwargs
```

Comment:
can we add a check to make sure `kwargs` consists of only primitive values that can be correctly serialized to json?